common/source/docs/common-cuav-pixhawkv6X.rst

```diff
@@ -0,0 +1,179 @@
+.. _common-cuav-pixhawkv6X:
+
+================
+CUAV Pixhawk v6X
+================
+
+
+.. image:: ../../../images/cuav-pixhawk-v6x.jpg
+
+
+Featuring STM32H7 cpu, vibration isolation of IMUs, redundant IMUs, double redundant barometers, IMU heating, and integrated Ethernet for high speed connections to companion computers.
+
+Where To Buy
+============
+
+The Pixhawkv6X flight controllers are sold by `CUAV <https://store.cuav.net/uav-flight-controller/>`__
+
+Specifications
+==============
+
+-  **Processor**
+
+   -  STM32H753IIK6
+   -  STM32F103
+
+
+-  **Sensors**
+
+   -  Bosh BMI088 IMU (accel, gyro)
+   -  InvenSense ICM-20649 IMU (accel, gyro)
+   -  InvenSense ICM-42688-P IMU (accel, gyro)
+   -  RM3100 magnetometer
+   -  Dual ICP-20100 barometers
+
+-  **Power**
+
+   -  Dual SMBUS/I2C Power Module Inputs
+   -  CAN Power Module included with autopilot
+
+-  **Interfaces**
+
+   -  8x UARTS, 6 Available for customer use
+   -  16x PWM outputs
+   -  PPM/SBUS input, DSM/SBUS input
+   -  SPI6 port
+   -  2x I2C ports for external compass, airspeed sensor, etc. on GPS connector
+   -  USB port (with remote cabling), USB connector on module
+   -  2 x CAN port
+   -  2 x CAN based Power Module inputs
+   -  Buzzer and Safety Switch
+   -  microSD card
+   -  Ethernet
+
+-  **Size and Dimensions**
+
+   - Heat disipating aluminum case
+   - tbd mm x tbd mm x tbd mm
+   - tbd g
+
+Typical System Wiring
+=====================
+
+.. image:: ../../../images/cuav-v6X-wiring.jpg
+    :target: ../../_images/cuav-v6X-wiring.jpg
+
+UART Mapping
+============
+
+ - SERIAL0 -> USB 
+ - SERIAL1 -> UART7 (Telem1) RTS/CTS pins
+ - SERIAL2 -> UART5 (Telem2) RTS/CTS pins
+ - SERIAL3 -> USART1 (GPS1)
+ - SERIAL4 -> UART8 (GPS2)
+ - SERIAL5 -> USART2 (Telem3) RTS/CTS pins
+ - SERIAL6 -> UART4 (User)
+ - SERIAL7 -> USART3 (Debug)
+ - SERIAL8 -> USB Virtual(MAVLink, can be used for SLCAN with protocol change)
+
+RC Input
+========
+The PPM pin, which by default is mapped to a timer input, can be used for all ArduPilot supported receiver protocols, except CRSF/ELRS and SRXL2 which require a true UART connection. However, FPort, when connected in this manner, will only provide RC without telemetry. 
```

Comment:
There is no SBUS inverter on the PPM interface, so it does not support SBUS. The DSM and SBUS receivers need to be connected to the DSM/SBUS interface

Reply:
unlike PX4, ArduPilot will decode all unidirectional RC protocols from the PPM pin: PPM, SBUS, DSM, IBUS, SRXL, the SBUS portion of FPort (not recommended, but it does work)....CRSF,SRXL2,ELRS need full UART access